### ISSUE TITLE
Test against video.js 6.0

### ISSUE THREAD

**Issue by thijstriemstra** [collab-project/videojs-wavesurfer#27]
6.0 was released recently.

**Comment by thijstriemstra** [comment]
Tested against 6.0.0 pre-release and at first glance it seems to work fine, only one warning:

```
VIDEOJS: WARN: videojs.plugin() is deprecated; use videojs.registerPlugin() instead
```

**Comment by thijstriemstra** [comment]
released 1.3.2 that fixes the warning. Porting videojs-wavesurfer to 6.0.0 'advanced plugins' can be done later.
